manifests/charts/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -24,7 +24,7 @@
     {{- end }}
   {{- end }}
 {{- end }}
-{{ $nativeSidecar := (eq (env "ENABLE_NATIVE_SIDECARS" "false") "true") }}
+{{ $nativeSidecar := (or (and (not (isset .ObjectMeta.Annotations `sidecar.istio.io/native-sidecar`)) (eq (env "ENABLE_NATIVE_SIDECARS" "false") "true")) (eq (index .ObjectMeta.Annotations `sidecar.istio.io/native-sidecar`) "true")) }}
```

Comment:
This is super ugly. Open to other suggestions...

Reply:
at least some newlines/indentation will go a long way :slightly_smiling_face: